requirements/base.txt

```diff
@@ -167,7 +168,7 @@ edx-drf-extensions==10.3.0
     # via
     #   -r requirements/base.in
     #   edx-rbac
-edx-enterprise-subsidy-client==0.4.2
+edx-enterprise-subsidy-client==0.4.3
```

Comment:
Not this change is in a [PR](https://github.com/openedx/edx-enterprise-subsidy-client/pull/83) and not currently deployed

Reply:
things are released now!